.github/workflows/zowe-cli-plugin.yml

```diff
@@ -40,18 +40,9 @@ jobs:
         echo "::remove-matcher owner=eslint-compact::"
         echo "::remove-matcher owner=eslint-stylish::"
 
-    - name: Use NPM v8
-      run: npm install -g npm@^8
-
     - name: Install Dependencies
       run: npm ci
 
-    - name: Update Dependencies
-      id: npm-update
-      uses: zowe-actions/octorelease-script@master
-      with:
-        script: npmUpdate
-

```

Comment:
We want to keep this step - updating npm to v8 isn't necessary with Node 16 and newer, but we still want to keep our dependencies up to date.

Reply:
The `zowe-actions/octorelease-script@master` should be `zowe-actions/octorelease/script@v1`